impeller/aiks/canvas.h

```diff
@@ -102,13 +102,15 @@ class Canvas {
 
   void DrawLine(const Point& p0, const Point& p1, const Paint& paint);
 
-  void DrawRect(Rect rect, const Paint& paint);
+  void DrawRect(const Rect& rect, const Paint& paint);
+
+  void DrawOval(const Rect& rect, const Paint& paint);
 
   void DrawRRect(Rect rect, Point corner_radii, const Paint& paint);
 
-  void DrawCircle(Point center, Scalar radius, const Paint& paint);
+  void DrawCircle(const Point& center, Scalar radius, const Paint& paint);
 
-  void DrawPoints(std::vector<Point>,
+  void DrawPoints(const std::vector<Point>& points,
```

Comment:
I think this should remain as `std::vector<Point> point`, though its not really enforced today we only call impeller::Canvas drawPoints with an r-value point vector, so the moves avoid an additional copy:

https://github.com/flutter/engine/blob/main/impeller/display_list/dl_dispatcher.cc#L869

Reply:
I reverted this, but holding off on a new commit until I figure out what to do about the other issues.